src/reactComponents/FileManageModal.tsx

```diff
@@ -441,6 +456,23 @@ export default function FileManageModal(props: FileManageModalProps) {
             onDoubleClick: () => handleRowDoubleClick(record),
           })}
         />
+        <br />        
+        <div style={{
```

Comment:
Can you put "Create a new {{type}}" here? If that's not possible, then "Create New" would be ok.